Decred Wallet/Features/Transactions/TransactionsViewController.swift

```diff
@@ -67,121 +70,115 @@ class TransactionHistoryViewController: UIViewController {
             self.refreshControl.endRefreshing()
             return
         }
-        
+
         self.allTransactions = txs
+
+        //setupTxSorter dropdown
+        let sorterOptions = [LocalizedStrings.newest,
+                             LocalizedStrings.oldest]
+        self.transactionSorterDropDown.initMenu(sorterOptions) { [weak self] index, value in
+            self?.reloadTxsForCurrentFilter()
+        }
+        self.transactionSorterDropDown.setSelectedItemIndex(0, callAct: false)
+
+        self.setupTxFilterDropDown()
+        self.transactionFilterDropDown.setSelectedItemIndex(0, callAct: false)
+
         self.transactionsTableView.backgroundView = nil
         self.transactionsTableView.separatorStyle = .singleLine
-        self.setupTxFilterAndDisplayAllTxs()
+
+        self.transactionsTableView.reloadData()
+        self.refreshControl.endRefreshing()
     }
     
-    func setupTxFilterAndDisplayAllTxs() {
-        var filterOptions = [LocalizedStrings.all.appending(" (\(self.allTransactions.count))")]
+    func setupTxFilterDropDown() {
+        var filterOptions = [LocalizedStrings.all]
         self.transactionFilters = [DcrlibwalletTxFilterAll]
-        
-        let sentCount = self.allTransactions.filter{$0.direction == DcrlibwalletTxDirectionSent}.count
+
+        let sentCount = self.allTransactions.filter {$0.direction == DcrlibwalletTxDirectionSent}.count
```

Comment:
Let's defer to dcrlibwallet to give us the counts for the different filters. Here's an idea:

- declare a dcrlibwallet count transactions helper function here:
```
let countTransactionsFor: (Int32) -> Int = { txFilter in
    let intPointer = UnsafeMutablePointer<Int>.allocate(capacity: 4)
    defer {
        intPointer.deallocate()
    }
    
    do {
        try WalletLoader.shared.firstWallet?.countTransactions(txFilter, ret0_: intPointer)
    } catch let error {
        print("count tx error:", error.localizedDescription)
    }

    return intPointer.pointee
}
```

- use the helper function to count transactions. note that you should pass in the txfilter, not txtype or direction.
```
if countTransactionsFor(DcrlibwalletTxFilterSent) > 0 {
    filterOptions.append(LocalizedStrings.sent)
    self.transactionFilters.append(DcrlibwalletTxFilterSent)
}
```